includes/query/SMW_Query.php

```diff
@@ -365,7 +365,8 @@ public function applyRestrictions() {
 			if ( count( $log ) > 0 ) {
 				$this->errors[] = Message::encode( array(
 					'smw_querytoolarge',
-					str_replace( '[', '&#x005B;', implode( ', ', $log ) )
+					str_replace( '[', '&#x005B;', implode( ', ', $log ) ),
+					count( $log )
 				) );
```

Comment:
@mwjames Well if one knows how to do it when it is actually not very hard at all.